### ISSUE TITLE
V1.28.2 introduces a bug causing our CI to fail due to bad requests

### ISSUE THREAD

**Issue by Osseta** [percy/cli#1568]
## The problem

V1.28.2 causes requests for assets that do not exists to be sent to our application server which causes the server to raise an error and our test suite ro fail.

This only occurs when we request a percy snapshot.

The error looks like

```
    ActionController::RoutingError:
       No route matches [GET] "/__serialized__/_fo6qheiye.svg+xml"
     # ./lib/static_asset_middleware.rb:14:in `call'
```

It appears related to the introduction `serializeBase64()` of  in [commit fe2a](https://github.com/percy/cli/commit/0a48e87d5b5cfaf1d469728e3d6de34171edfe2a) 

## Environment

- Node version:
- `@percy/cli` version: 1.28.2
- Version of Percy SDK you’re using: percy-capybara 5.0.0
- OS version: Linux & MacOS
- Type of shell command-line [interface]: bash & zsh

## Debug logs

[debug.log](https://github.com/percy/cli/files/14926457/debug.log)

**Comment by prklm10** [comment]
We will look into this.

**Comment by pankaj443** [comment]
Hey @Osseta , a fix for this is released in the latest version, 1.28.3

**Comment by Osseta** [comment]
I can confirm that release 1.28.3 resolves the issue in our CI pipeline.
